### ISSUE TITLE
SA1003 triggers for null forgiving operator in front of array access operator

### ISSUE THREAD

**Issue by DWIAltonaAnalytics** [bjornhellander/NewStyleCopAnalyzers#12]
this triggers SA1003:
```cs
var foo = someNullableArrayOrDictionary![ArrayOrDictionaryAccess];
```
with error `Operator '!' should be followed by whitespace.`

but there is now whitespace allowed for `Array[Accessor]` so we have a rule conflict here.

**Comment by bjornhellander** [comment]
Fixed in version 1.0.1
